src/version.c

```diff
@@ -161,6 +162,17 @@ int check_versions(int *current_version,
 		return -1;
 	}
 
+	if (requested_version != -1) {
+		if (requested_version <= *current_version) {
+			fprintf(stderr, "Requested version for update (%d) must be greater than current version (%d)\n",
+				requested_version, *current_version);
+			return -1;
+		}
+		if (requested_version < *server_version) {
+			*server_version = requested_version;
+		}
+	}
```

Comment:
Just so I make sure I'm understanding, if `requested_version` is greater than `server_version` you just get the `server_version`. That's good, should it print something out that you *can't* go to the requested version?

Reply:
It is correct. It also works that way to not get on the way of updates that involved multiple formats (and re-executes). Because of that, unless we do more work, there's no clear place to print this -- sometimes we CAN go to the requested version, if we re-execute.

Let's say 100 -> 190 are in format 1, and 200 -> 280 are in format 2. The bump is 190-200. And we start from 150.

- `-m 140` fails because we can't rollback
- `-m 150` fails because it would be a nop
- `-m 160` goes to 160 since it is < than server version (190).
- `-m 250` goes to server_version (190), and then re-executes.
- `-m 200` goes to server_version (190), and doesn't re-execute, since going to 190, it sees itself as 200, and that is what was requested
- `-m 300` goes to server version (190), re-executes, goes to new server version (280), and that is it.

Reply:
We could add a note if we are not re-executing and requested_version > server_version (after the reexec check says it is not reexecing). I'm not sure how useful it is since we already explicit state in the output to what version we are really going.

Reply:
> we already explicit state in the output to what version we are really going.

Ok